src/types/nitro.ts

```diff
@@ -270,12 +272,23 @@ export interface NitroOptions extends PresetOptions {
   renderer?: string;
   serveStatic: boolean | "node" | "deno" | "inline";
   noPublicDir: boolean;
+
   /**
    * @experimental Requires `experimental.wasm` to work
    *
    * @see https://github.com/unjs/unwasm
    */
   wasm?: UnwasmPluginOptions;
+  openAPI?: {
+    meta?: {
+      title?: string;
+      description?: string;
+      version?: string;
+    };
+    ui?: {
+      scalar?: ReferenceConfiguration;
+    };
+  };
```

Comment:
```suggestion
  openAPI?: {
    meta?: {
      title?: string;
      description?: string;
      version?: string;
    };
    ui?: {
      scalar?: ReferenceConfiguration;
    };
  } | boolean;
```

Shouldn't we add `boolean` to interface since it still supports `false` and `true` ?

Reply:
i don't think it would be best. We might add `enabled` field specially because later it needs to be configurable per env (dev and prod) and not having mixed types makes relying on types easier.